### ISSUE TITLE
LAN system overhaul

### ISSUE THREAD

**Issue by ehne** [Double-Fine-Game-Club/pongball#21]
TL;DR,  No IP addresses, connection dialogs, or decisions about Hosting or Joining

I'll be reworking the lan system to be more streamlined by using captainMess.
pretty much i'll be doing:
- [ ] use only one button for match-making (over wifi, no need to 'host' and 'join' games)
- [ ] making it look pretty

feel free to add suggestions!

Edit: 
Currently we have a simple unity networkingsystem that requires one user to host the game and the other to join, This approach works well, but means that the user has to do some 'troubleshooting' before they can begin. Whereas if we used captainsMess, both users would have to press 'LAN play' and they would be in the game, without having to 'troubleshoot'. This would result in a much cleaner and smoother lan system.

-darcy

**Comment by Cheeseness** [comment]
@bobsayshilol @lightsoda and @jctwood have been poking around with networking related stuff in some form or another. It'd be good to get their thoughts and perspectives along the way.

Edit: I think it'd also be good to explain some pros/cons of what we have now vs [CaptainsMess](https://github.com/hengineer/CaptainsMess)

**Comment by ehne** [comment]
seems a bit stupid to do all of that for one bit of the game
